src/test/java/SetTest.java

```diff
@@ -0,0 +1,43 @@
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("컬렉션_테스트")
+public class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        this.numbers = new HashSet<>();
+        this.numbers.add(1);
+        this.numbers.add(1);
+        this.numbers.add(2);
+        this.numbers.add(3);
+    }
+
+    @Test
+    void 컬렉션_크기_구하기() {
+        int size = this.numbers.size();
+
+        assertThat(size).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void 숫자_포함여부_확인(int value) {
+        assertThat(this.numbers.contains(value)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(input);
```

collection의 경우 위와 같이 구현도 가능함